lib/membrane_rtmp_plugin/rtmp/source/messages/command/connect.ex

```diff
@@ -0,0 +1,66 @@
+defmodule Membrane.RTMP.Messages.Connect do
+  @moduledoc false
+
+  @behaviour Membrane.RTMP.Message
+
+  alias Membrane.RTMP.AMFEncoder
+
+  @enforce_keys ~w(app type supports_go_away flash_version swf_url tc_url)a
+  defstruct [tx_id: 0] ++ @enforce_keys
```

Comment:
There were some comments about this already, please make sure that correct style applies everywhere